src/main/java/org/gradle/profiler/mutations/support/FileSupport.java

```diff
@@ -0,0 +1,27 @@
+package org.gradle.profiler.mutations.support;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.OpenOption;
+import java.nio.file.Path;
+
+public class FileSupport {
+
+    public static void writeUnchecked(Path path, String text, OpenOption... options) {
+        try {
+            Files.write(path, text.getBytes(StandardCharsets.UTF_8), options);
+        } catch (IOException e) {
+            throw new UncheckedIOException("Could not write content to a file: " + path, e);
+        }
+    }
+
+    public static String readUnchecked(Path path) {
+        try {
+            return new String(Files.readAllBytes(path), StandardCharsets.UTF_8.toString());
+        } catch (IOException e) {
+            throw new UncheckedIOException("Could not read contents of a file: " + path, e);
+        }
+    }
+}
```

Comment:
I'm wondering if these `*Support` types are worth it vs. inlining the trivial code, or using something already existing. Let's leave it like this for now, but let's talk about this during the retro or something.

Reply:
They are mostly here since it makes code much cleaner and easier to read. For that particular example, always handling these exceptions to throw unchecked exceptions makes a lot of noise.